### PR TITLE
[CAMEL-17954]fix integration tests failure

### PR DESCRIPTION
**Pull request by ffang**

### CODE REVIEW

File: components/camel-salesforce/it/resources/CamelSalesforceIntegrationTests.connectedApp [hunk 1]
```diff
@@ -21,8 +21,8 @@
...
-        <consumerKey>${salesforce.client.id}</consumerKey>
-        <consumerSecret>${salesforce.client.secret}</consumerSecret>
+        <!--consumerKey>${salesforce.client.id}</consumerKey>
+        <consumerSecret>${salesforce.client.secret}</consumerSecret-->
```
Comment (zhfeng) [inline components/camel-salesforce/it/resources/CamelSalesforceIntegrationTests.connectedApp:25]: Is there any reason to comment these two propreties? they are not used in the test?

Comment (ffang) [reply components/camel-salesforce/it/resources/CamelSalesforceIntegrationTests.connectedApp:25]: Hi @zhfeng ,

No, they are not used in the test, the test will pick up the salesforce.client.id and salesforce.client.secret properties from test-salesforce-login.properties directly.

Plus, having consumerKey and consumerSecret in CamelSalesforceIntegrationTests.connectedApp actually breaks the latest version of ant migration tool(ant-salesforce.jar complains those and failed). Only removing those can setup the salesforce server and run the integration test.

Cheers
Freeman

Comment (zhfeng) [reply components/camel-salesforce/it/resources/CamelSalesforceIntegrationTests.connectedApp:25]: So it should be better to remove them but not just comment. Or add some words to explain this reason.

File: components/camel-salesforce/camel-salesforce-component/src/test/java/org/apache/camel/component/salesforce/CompoundTypesIntegrationTest.java [hunk 2]
```diff
@@ -23,8 +23,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.junit.Assert.assertEquals;
```
Comment (jeremyross) [inline components/camel-salesforce/camel-salesforce-component/src/test/java/org/apache/camel/component/salesforce/CompoundTypesIntegrationTest.java:26]: Please use assertions from org.junit.jupiter.api.Assertions.

Comment (ffang) [reply components/camel-salesforce/camel-salesforce-component/src/test/java/org/apache/camel/component/salesforce/CompoundTypesIntegrationTest.java]: Good catch, revised accordingly.

Thanks @jeremyross !

**Review by davsclaus — APPROVED** [review]

**Comment by jeremyross** [comment]
Merged.

**Pull request closed without merging**